<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SuperCellDialog</class>
 <widget class="QDialog" name="SuperCellDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Super Cell Parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="3">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Super Cell Options</string>
       </property>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="aRepeatLabel">
          <property name="text">
           <string>A repeat:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="aCellSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="bRepeatLabel">
          <property name="text">
           <string>B repeat:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="bCellSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="cRepeaLabel">
          <property name="text">
           <string>C repeat:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="cCellSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="generateCell">
       <property name="text">
        <string>Generate Cell</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SuperCellDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
